src/components/Common/CaseStudyCards/CaseStudyLayout.js

```diff
@@ -15,6 +16,20 @@ const FlexCol = styled(Col)`
 `
 
 const RowLayout = styled(Row)`
+  ${breakpoint('smallPhone')`
+    padding-top: ${props =>
+      props.isTop ? props.theme.spacing[3] : props.theme.spacing[3.5]};
+    padding-bottom: ${props =>
+      props.isTop ? props.theme.spacing[3] : props.theme.spacing[3.5]};
+  `}
+
+  ${breakpoint('tablet')`
+    padding-top: ${props =>
+      props.isTop ? props.theme.spacing[3.5] : props.theme.spacing[5]};
+    padding-bottom: ${props =>
+      props.isTop ? props.theme.spacing[5] : props.theme.spacing[5]};
```

Comment:
do we need this ternary?

Reply:
Oops. Good point. Just pushed the correct top & bottom paddings for "top" CaseStudy cards. Gonna look over this with Antonas when he is ready to review, because these values don't exist in `spacing` in our theme. So gotta check